data/drives.txt

```diff
@@ -0,0 +1,58 @@
+"Hai Hyperdrive"
+	category "Systems"
+	cost 80000
+	thumbnail "outfit/hyperdrive"
+	"mass" 15
+	"outfit space" -10
+	"hyperdrive fuel" 80
+	"jump speed" .3
+	"hyperdrive" 1
+	description "This mysterious box sits quietly at the heart of your ship, thinking its own inscrutable thoughts. When it comes time to travel between the stars it sets off a nuclear fusion reaction powerful enough to level an entire city, and then somehow tucks the resulting explosion in between the folds of regular space-time like a wasp stinging an oak leaf to lay her eggs inside. Your ship rides the resulting ripple in the fabric of space."
+	description "	They say that fewer than a dozen human beings alive today understand the inner workings of the hyperdrive. This one is a Hai version that is more efficient as a regular human one."
+
+"Korath Hyperdrive"
+	category "Systems"
+	cost 80000
+	thumbnail "outfit/hyperdrive"
+	"mass" 15
+	"outfit space" -10
+	"hyperdrive fuel" 120
+	"jump speed" .4
+	"hyperdrive" 1
+	description "This mysterious box sits quietly at the heart of your ship, thinking its own inscrutable thoughts. When it comes time to travel between the stars it sets off a nuclear fusion reaction powerful enough to level an entire city, and then somehow tucks the resulting explosion in between the folds of regular space-time like a wasp stinging an oak leaf to lay her eggs inside. Your ship rides the resulting ripple in the fabric of space."
+	description "	They say that fewer than a dozen human beings alive today understand the inner workings of the hyperdrive. This one is a Korath version faster but consumes a bit more fuel."
```

Comment:
This one is a Korath version - faster, and takes less space, but consumes a bit more fuel."